app/controllers/users/invitations_controller.rb

```diff
@@ -13,19 +13,24 @@ class Users::InvitationsController < Devise::InvitationsController
     # If an anonymous user tries to send an invitation, send them to the invitation page after sign-in.
     require_sign_in(redirect_after_login: new_user_invitation_path)
   end
+  before_action :load_vita_partners, only: [:new]
```

Comment:
you might be able to replace this and the private method with
```ruby
load_resource :vita_partner, collection: [:new], parent: false, only: [:new]
```

Reply:
@bengolder we tried this method but got 403 errors